wai-aria/scripts/aria-utils.js

```diff
@@ -55,6 +55,66 @@ const AriaUtils = {
   },
 
 
```

Comment:
Adding some notes to ease reviewers' understanding of the PR.

I've added a new `verifyRoleOrVariantRolesBySelector` function that asserts true if the element's computed role matches any of the provided roles, or false if none match.

For now, it's only called by the `verifyGenericRolesBySelector` helper function below, because WebKit returns an empty computed role string for generic elements and Chromium returns "none"… The ARIA WG discussed and agreed it would be undesirable to ask for implementation changes that result in churn with no meaningful user impact.

So this allows the HTML-AAM `generic` role tests to pass in all implementations, despite some minor acceptable differences in the output of computedrole for generics.